internal/pkg/template/input/validation.go

```diff
@@ -24,31 +24,53 @@ func validateInputDefault(fl goValidator.FieldLevel) bool {
 	}
 
 	// Check if Default is present in Options
-	if fl.Parent().FieldByName("Kind").String() == "select" || fl.Parent().FieldByName("Kind").String() == "multiselect" {
-		for _, x := range fl.Parent().FieldByName("Options").Interface().([]Option) {
-			if string(x) == fl.Field().String() {
-				return true
+	if fl.Parent().FieldByName("Kind").String() == KindSelect {
+		return findValueInOptions(fl.Field().String(), fl)
+	}
+
+	if fl.Parent().FieldByName("Kind").String() == KindMultiSelect {
+		defaultValues := fl.Field().Interface().([]string)
+		for _, defaultValue := range defaultValues {
+			found := findValueInOptions(defaultValue, fl)
+			if !found {
+				return false
 			}
 		}
-		return false
+		return true
+	}
+
+	if fl.Parent().FieldByName("Kind").String() == KindInput && fl.Parent().FieldByName("Type").String() != "" {
+		err := validateUserInputByType(fl.Field(), fl.Parent().FieldByName("Type").String())
+		if err != nil {
+			return false
+		}
 	}
 
 	// Check that the Default has the right Type for the Kind
-	err := validateUserInputTypeByKind(fl.Field(), fl.Field().Kind().String())
+	err := validateUserInputTypeByKind(fl.Field(), fl.Parent().FieldByName("Kind").String())
```

Comment:
Tohle mi připomnělo že jsem se chtěl předtím zeptat jestli ty `Type` a `Kind` v definici inputu nepřejmenovat, protože `reflect` a z něj celý ten Validator  používá `Kind` pro typ a dělá to v tom kódu trošku zmatek. A je to náchylný na takovýhle chybky. Ale teď už je teď asi jedno. 😃